extras/create_snapshots_reports_scorecard.py

```diff
@@ -288,30 +288,41 @@ def create_snapshot(db, cyhy_db_section, org_id, use_only_existing_snapshots):
             org_descendants = db.RequestDoc.get_all_descendants(org_id)
 
     if snapshot_process.returncode == 0:
-        logging.info("Successful snapshot: %s (%.2f s)", org_id, snapshot_duration)
+        logging.info("[%s] Successful snapshot: %s (%.2f s)", threading.current_thread().name, org_id, snapshot_duration)
         with ss_lock:
             successful_snapshots.append(org_id)
             if org_descendants and not use_only_existing_snapshots:
                 logging.info(
-                    " - Includes successful descendant snapshot(s): %s", org_descendants
+                    "[%s]  - Includes successful descendant snapshot(s): %s", threading.current_thread().name, org_descendants
                 )
                 successful_snapshots.extend(org_descendants)
     else:
-        logging.error("Unsuccessful snapshot: %s", org_id)
+        logging.error("[%s] Unsuccessful snapshot: %s", threading.current_thread().name, org_id)
         with fs_lock:
             failed_snapshots.append(org_id)
             if org_descendants and not use_only_existing_snapshots:
                 logging.error(
-                    " - Unsuccessful descendant snapshot(s): %s", org_descendants,
+                    "[%s]  - Unsuccessful descendant snapshot(s): %s", threading.current_thread().name, org_descendants,
                 )
                 failed_snapshots.extend(org_descendants)
-        logging.error("Stderr failure detail: %s %s", data, err)
+        logging.error("[%s] Stderr failure detail: %s %s", threading.current_thread().name, data, err)
     return snapshot_process.returncode
 
 
-def create_snapshots_from_list(org_list, db, cyhy_db_section):
+def create_all_snapshots(db, cyhy_db_section):
     """Create a snapshot for each organization in a list."""
```

Comment:
Not a big fan of the new function name but more importantly I would like the function docstring to better reflect what the function is now doing. I think the name is unappealing because it doesn't really reflect the threaded use of this function. Since we're no longer passing in a list of orgs I think the docstring should make some reference to the global list being consumed.

Reply:
That's fair.  Please have a look at https://github.com/cisagov/cyhy-reports/pull/106/commits/e969421ea40668481a76a06786db823b27d54eb9 and let me know what you think.  Once we come to consensus on this, I will make similar changes for the report generation functions (which will also resolve #59).

Reply:
Ah yes, that reads much better. Thank you for being comprehensive about your approach!

Reply:
@dav3r - I have one minor quibble.  Please see [here](https://github.com/cisagov/cyhy-reports/pull/106#pullrequestreview-1854813077).

Reply:
Done and done. 👍

Reply:
As mentioned earlier in this conversation, please see https://github.com/cisagov/cyhy-reports/pull/106/commits/c410e90e6457f84bfefe22006790ca0e7a91d910 where I cleaned up the report generation functions along the same lines as what I did in https://github.com/cisagov/cyhy-reports/commit/e969421ea40668481a76a06786db823b27d54eb9.

@mcdonnnj I think we can now resolve this conversation if you're satisfied.